src/main/scala/cognite/spark/v1/AlphaDataModelInstancesHelper.scala

```diff
@@ -347,5 +351,31 @@ object AlphaDataModelInstancesHelper {
       case _ =>
         throw SparkSchemaHelperRuntime.badRowError(row, keyString, "String", "")
     }
+
+  def getDirectRelationIdentifierProperty(
+      externalId: String,
+      row: Row,
+      keyString: String,
+      index: Int): DirectRelationIdentifier =
+    row.get(index) match {
+      case identifier: String =>
+        // Colon character cannot be used for space but it can be used for the externalId, so we should consider only
```

Comment:
We should have a test for that case, then. I still don't like that it's allowed, though...